Makefile

```diff
@@ -1,27 +1,40 @@
 BIN = mackerel-agent
 ARGS = "-conf=mackerel-agent.conf"
 
-all: clean build test
+BUILD_FLAGS = "\
```

Comment:
Might be renamed to BUILD_LDFLAGS or something as they're not flags for `go build`, but `go build -ldflags=`.